pom.xml

```diff
@@ -233,6 +233,12 @@
             <scope>compile</scope>
         </dependency>
 
+        <dependency>
```

Comment:
Not sure how this was tested, but `grails run-app` is now broken (because the dependency is present only for maven).  Happily, it seems as though [there is a prettytime plugin](http://grails.org/plugin/pretty-time), so should be easy enough to pull it in that way (in `BuildConfig.groovy`).